site/src/components/CustomLogo/CustomLogo.tsx

```diff
@@ -24,8 +24,7 @@ export const CustomLogo: FC<{ css?: Interpolation<Theme> }> = (props) => {
 			onLoad={(e) => {
 				e.currentTarget.style.display = "inline";
 			}}
-			css={{ maxWidth: 200 }}
-			className="application-logo"
+			className="application-logo max-w-50"
```

Comment:
annoying quirk here: we're still on tailwind 3, which will not generate any code for this class. the v3 jit only recognizes certain fixed numbers.

https://v3.tailwindcss.com/docs/width